app/controllers/courses_controller.rb

```diff
@@ -147,6 +149,13 @@ def suspected_plagiarism
   # User-initiated actions #
   ##########################
 
+  # /courses/:slug/students/add_to_watchlist
```

Comment:
I would rather have this action in a separate sub-controller, similar to the `Courses::SyllabusesController`. The main Courses controller is already responsible for too many things.

Reply:
Okay, I will make the necessary changes